docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+
+services:
+  nginx:
+    image: nginx:1.15.3
+    depends_on:
+      - fixmystreet
+    ports:
+      - "8000:80"
+    volumes:
+      - ./conf/nginx.conf-docker:/etc/nginx/conf.d/default.conf
+    networks:
+      default:
+        aliases:
+          - nginx.svc
+  fixmystreet:
+    image: sagepe/fixmystreet:0.5
+    tty: true
+    depends_on:
+      - postgres
+      - memcached
+    volumes:
+      - ./conf/general.yml-docker:/var/www/fixmystreet/fixmystreet/conf/general.yml
```

Comment:
Not for now, perhaps, but it would be good to be able to generate this from the example so we don't have to remember to keep it up to date.

Reply:
Yes, good idea.

Another possibility to consider is allowing the environment variables to override the defaults at start-up perhaps, so we don't need to maintain a separate file at all.

Reply:
The current file is essentially an example of how you provide your own `general.yml` file - that's partly what I intended it to be, anyway.